chains/subnets_test.go

```diff
@@ -117,16 +117,12 @@ func TestSubnetConfigs(t *testing.T) {
 			config: map[ids.ID]subnets.Config{
 				constants.PrimaryNetworkID: {},
 				testSubnetID: {
-					GossipConfig: subnets.GossipConfig{
-						AcceptedFrontierValidatorSize: 123456789,
-					},
+					ValidatorOnly: true,
```

Comment:
This test wouldn't be meaningful id the subnet config was equal to the primary network config. Because we're deleting the only field that was different, we needed to introduce some other config value to be different.